elastic/java-ecs-logging
# ci(jenkins): enable CI/CD pipeline

## Highlights
- Enable CI/CD pipeline
- Enable JJBB creation
- Stages: build, test, docs, and sanity-check
- Sanity check will evaluate the below hooks using the [pre-commit](https://pre-commit.com/)
  - check-case-conflict
  - check-executables-have-shebangs
  - check-merge-conflict
  - check-bash-syntax
  - check-jenkins-pipelines
  - check-jjbb
